docs/tutorial-solver.md

```diff
@@ -56,15 +50,7 @@ Technically, the orderbook file can be saved anywhere, but we have chosen `targe
 
 ## Run the Solver
 
-First run and enter the solver container:
-
-```sh
-docker-compose run -v $PWD/:/app/dex-services stablex
-```
-
-Since the project was mounted inside the container, changes you make to the driver code will be directly reflected on every restart.
-
-The driver can now be run from within the Docker container using the command:
+From within the `dex-services` root directory,
 
 ```sh
 cargo run --bin driver -- --solver-type OpenSolver --node-url $NODE_URL --private-key $PRIVATE_KEY --orderbook-file $ORDERBOOK_FILE
```

Comment:
What I mean is that this could just be `cargo run --bin driver` passing everything as env vars only.

Reply:
But either way it's not very important. It just looked like an accident to me that we specify the options as both env vars and arguments.

Reply:
Right t this is described exactly in this way just a few lines below.

Reply:
I wanted to show the user that this is kind of a minimal necessary set of arguments and then propose an easier alternative for other configurations.